xtask/src/main.rs

```diff
@@ -397,7 +397,7 @@ fn test(build_params: &BuildParams) -> Result<()> {
     cmd.current_dir(workspace());
     cmd.arg("test");
     cmd.arg("--workspace");
-    cmd.arg("--config").arg("build.rustflags='--cfg platform=\"virt\"'");
+    cmd.arg("--target").arg("x86_64-unknown-linux-gnu");
```

Comment:
why is this change necessary?

Reply:
because the tests rely on the stdlib which the default targets that are now set in `Cargo.toml` are not providing. sadly i've found the tests generally only seem to work on `x86_64-unknown-linux-gnu` (sad macbook noises)

i removed the `--config` arg since it is not required (this fallback is already part of the `cfg` macro in the code